src/components/promo-banner/promo-banner.tsx

```diff
@@ -0,0 +1,26 @@
+import promoImage from '../../assets/promoScissors.svg';
+import Image from 'next/image';
+
+export const PromoBanner = () => {
+  return (
+    <div className="tracking-tigh sm:text-4xl uppercase font-rex w-48 mt-7 text-transform: skew-y-15">
+      <h1 className="text-3xl text-white leading-8">
+        Oldboy <br></br> barbershop
+      </h1>
+      <div className="text-orange-title text-base relative">
+        <p className="w-1/2 bg-orange-title h-1 absolute bottom-3 left-24"></p>
+        Крупнейшая сеть
+        <br></br> барбершопов
+      </div>
+      <div className="flex">
+        <span className="text-6xl text-white font-rex_inline leading-10 mt-4 ">294</span>
```

Comment:
I am pretty confident that `294` is dynamic. For now, we don't know where it comes from but we at least will put into props.

Define prop type like so and understand the code below:

```
interface PromoBannerProps {
  barbershopCount: number
 }
 
 Then:
 export const PromoBanner = ({ barbershopCount }: PromoBannerProps ) => {....}
 ```
 
 Congrats, you've just created your first TS type in this project